blockly-core/core/code_generation/generator.js

```diff
@@ -197,6 +197,11 @@ Blockly.CodeGenerator.prototype.blockToCode = function(block, opt_showHidden) {
   if (!block) {
     return '';
   }
+  if (block.skipGeneration) {
+    // Clear the flag, so it doesn't persist to future serializations.
+    block.skipGeneration = false;
```

Comment:
Not sure.  Definitely open to suggestions.  FWIW `skipGeneration` is a flag I added in this PR, and is used nowhere else.

What I'm trying to do is let a block own code generation for all descendant blocks connected to `nextConnection` (so it can be wrapped inside a callback).